### ISSUE TITLE
LeadsFilter

### ISSUE THREAD

**Issue by ramik5** [amocrm/amocrm-api-php#346]
Добрый день! Как отфильтровать сделки по незаполненному пользовательскому полю типа "дата"?

1) Выдает сделки в которых поле дата c отрицательным значением в timestamp
`$filter->setCustomFieldsValues([$fieldId=> '']);`

2) Игнорирует фильтр вовсе
`$filter->setCustomFieldsValues([$fieldId=> null]);`

**Comment by makeroi** [comment]
@ramik5 а вы подключены к альфа тестированию апи фильтрации?

**Comment by ramik5** [comment]
@makeroi да, остальные фильтры отрабатывают корректно, в том числе и на диапазон значений.

**Comment by eugene-borovov** [comment]
Незаполненное поле равно NULL? А во всех методах API значение NULL просто игнорируется в запросе (по крайней мере в этой библиотеке), даже выставить в NULL нельзя. Поэтому никак.

**Comment by ramik5** [comment]
@bessudnov Приветствую! Никита, прокомментируй пожалуйста.
